kpi/tests/api/v2/test_api_submissions.py

```diff
@@ -571,8 +567,7 @@ def test_get_legacy_edit_link_submission_owner(self):
         assert response.status_code == status.HTTP_200_OK
 
         expected_response = {
-            'url': 'http://server.mock/enketo/{}'.format(self.submission.get(
-                self.asset.deployment.INSTANCE_ID_FIELDNAME))
+            'url': 'http://server.mock/enketo/{}'.format(self.submission['_id'])
```

Comment:
For future reference:
 Before, the mock submissions where using `id` while MongoDB uses `_id` . The idea of `INSTANCE_ID_FIELDNAME` was to be able to share the same code between `*DeploymentBackend` classes without messing with `id` or `_id`. Over the time, we started to change the fixtures and mock data to use `_id` too. 
Since now use MockMongo, which behaves like MongoDB, we don't need this trick anymore. 

So we got rid of all of these kind of calls in favor of `['_id']`.